packages/datadog-plugin-fs/src/index.js

```diff
@@ -315,7 +315,7 @@ function makeFSFlagTags (resourceName, path, options, defaultFlag, config, trace
 }
 
 function makeFSTags (resourceName, path, options, config, tracer) {
-  path = options && 'fd' in options ? options.fd : path
+  path = options && typeof options === 'object' && 'fd' in options ? options.fd : path
```

Comment:
The 2 checks could be avoided by doing `options && options.fd` instead of `options && typeof options === 'object' && 'fd' in options` since the former supports any type. This is just a different syntax with the same meaning however, so non-blocking.

Reply:
It's not *quite* the same meaning here, because I do explicitly only want this code path for `'object'`. For anything that's not an object but still truthy, we just want the value, regardless if someone has attached a weird `fd` property to the wrapper prototype.